app/models/moderator_action.rb

```diff
@@ -150,6 +160,7 @@ def set_resource_user_id
 
   def set_resource_content
     return unless resource
+    return if action == RENAME
```

Comment:
The benefit of performing this task with a ModeratorAction is to preserve some history, so you should probably set `resource_content` to something like `resource.try_methods( :name, :title )` here.